Core/Rewriters/OrderedMockRewriter.cs

```diff
@@ -54,16 +56,20 @@ private IEnumerable<SyntaxNode> ReplaceExpressionStatements (SyntaxList<Statemen
         }
 
         var firstIdentifierName = statement.GetFirstIdentifierName();
-        var newExpressionStatement = statement.ReplaceNode (
+        var newExpressionStatement = (ExpressionStatementSyntax) statement.ReplaceNode (
             firstIdentifierName,
-            MoqSyntaxFactory.InSequenceExpression (firstIdentifierName, current.ToString()));
+            MoqSyntaxFactory.InSequenceExpression (firstIdentifierName, currentIndex));
```

Comment:
`InSequenceExpression` should rather accept `int?`, this would make things way more understandable on this side of the code, instead of passing in an index as a string (possibly being empty).